addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/PolicyTag.java

```diff
@@ -35,6 +35,13 @@ public enum PolicyTag {
     QA_STD("QA Standard.policy", "QA Standard", "scanpolicies"),
     QA_FULL("QA Full.policy", "QA Full", "scanpolicies"),
     API("API.policy", "API", "scanpolicies"),
+    /**
+     * For rules believed to be of interest to Penetration Testers. Essentially everything other
+     * than Example rules. Added originally for Passive Rules.
```

Comment:
Do we need the passive remark?

Reply:
I guess not. I was mainly thinking something like: "If I don't get back to applying this to active rules will someone be confused in two years?" (Which I felt was a "yes")

But I'm not overly set on it, I can trim/drop.

Reply:
I'd drop it.